build/tools/build_parallel.js

```diff
@@ -0,0 +1,203 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+'use strict';
+
+var path = require('path');
+var fs = require('fs');
+var findRoot = require('find-root');
+var util = require('util');
+var runAll = require('npm-run-all');
+var async = require('async');
+var EventEmitter = require('events');
+
+// See section titled "About MaxListenersExceededWarning" in https://github.com/mysticatea/npm-run-all/blob/HEAD/docs/node-api.md
+EventEmitter.defaultMaxListeners = 30;
+
+var usage = function() {
+  console.log('usage: node ' + __filename + ' task');
+  console.log('where task is one of:');
+  var tasks = require('./projects.json').tasks;
```

Comment:
>var tasks = require('./projects.json').tasks; [](start = 2, length = 45)

that is going to make a call to usage() block while the file is loaded - not that big of a deal but why not require it at the top with all the other require statements?